charts/cluster-api-provider-openshift-assisted/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: cluster-api-provider-openshift-assisted
+description: Cluster API Bootstrap and Controlplane providers for OpenShift Assisted Installer
+type: application
+version: "4.19"
+appVersion: "4.19"
```

Comment:
How is this version used? Wouldn't be helpful to store actual versions instead of OCP version?

Reply:
The other providers _are_ versioned based on OCP so it makes at least a bit of sense to use it for them.
For us I think we should just stick to the MCE version since that's likely what we'll have a branch for in our repo.